src/index.js

```diff
@@ -0,0 +1,11 @@
+import { listItems, listLikes } from './modules/listItems.js';
+import './style.css';
+
+window.onload = () => {
+  listItems();
+  listLikes();
+};
+
+window.addEventListener('load', () => {
+  listLikes();
+});
```

Comment:
- [ ] [OPTOINAL] Hi, it appears that these lines of code are unnecessary. you already have a `window.onload` function that calls `listLikes()`. which means you are calling list likes twice.
```suggestion
```

Reply:
although this was for the refresh button on the browser, I deleted it. thanks